platformio.ini

```diff
@@ -16,9 +16,10 @@ monitor_speed = 115200
 monitor_raw = True
 lib_deps = 
 	arduino-libraries/NTPClient@^3.1.0
+	https://github.com/tzapu/WiFiManager.git
 	paulstoffregen/Time@^1.6.1
 	jchristensen/Timezone@^1.2.4
-	tzapu/WiFiManager@^0.16.0
+	;tzapu/WiFiManager@^0.16.0
```

Comment:
Whats the difference between these dependencies changes?

Reply:
it should be the same, but the original did't show some of the functions. so I used the other option and it worked, could try to return to the oter one.

Reply:
Ok, so I think we should remove line 22 then.
```suggestion
```